CMakeLists.txt

```diff
@@ -96,6 +83,59 @@ set_property(CACHE STRICT_GSL_CHECKS PROPERTY STRINGS ${STRICT_GSL_CHECKS_Values
 
 if (WIN32)
 	option(MSI_REDISTRIBUTE_UCRT_NONASL "Redistribute Universal C Runtime DLLs with the MSI generated by CPack. The resulting MSI is not distributable under Apache 2.0." OFF)
+	option(ENABLE_WEL "Enables the suite of Windows Event Log extensions." OFF)
+	option(ENABLE_PDH "Enables PDH support." OFF)
+endif()
+
+if(CMAKE_SYSTEM_NAME STREQUAL "Linux")
+	option(ENABLE_SYSTEMD "Disables the systemd extension." ON)
```

Comment:
the description should say "Enables ..."

Reply:
fixed in eaf13e4